tests/unit/cli/inspect/test_code_list_inspector.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+
+from csvcubed.models.sparqlresults import CatalogMetadataResult, CodeListTableIdentifers
+from csvcubed.utils.sparql_handler.code_list_inspector import CodeListInspector
+from csvcubed.utils.tableschema import CsvwRdfManager
+from tests.unit.test_baseunit import get_test_cases_dir
+
+_test_case_base_dir = (
```

Comment:
I think you should alter the base dir to be up to the `inspect` dir so it can be referenced in all of the tests.